.circleci/config.yml

```diff
@@ -95,43 +95,7 @@ jobs:
               done
               exit $ret
             fi
-  build-push-docker-rust-code:
-    executor: ubuntu-2xl
-    parameters:
-      image_target:
-        description: what value to pass to `IMAGE_TARGET=<value>` when running the docker/build-common.sh script which runs the Rust build.
-        type: string
-        default: NOT_PROVIDED
-    steps:
-      - checkout
-      - aws-ecr-setup
-      - run: echo $GITHUB_CONTAINER_REGISTRY_TOKEN | docker login ghcr.io -u USERNAME --password-stdin
-      - run:
-          name: Build and Push
-          shell: /bin/bash
-          environment:
-            IMAGE_TARGET: << parameters.image_target >>
-          command: |
-            set -e
-            docker buildx create --use
-            docker/docker-bake-rust-all.sh
-  build-push-community-platform:
```

Comment:
I agree with removing `build-push-docker-rust-code` above, but should we keep the community platform build going in CircleCI as well while we bake the new GHA workflows so we have a backup?

Reply:
hmm, I'm relatively confident that the GHA workflows will work reliably enough to build the community-platform image (also the corresponding circleci workflow i just 2 weeks old and already uses bake). The problem is that both use the same docker-bake file and if we have both the circle and GHA workflow running in parallel they will concurrently write to the same Artifact Repository (with the same tagging scheme).

Reply:
if you want some safety backup, I just comment out the circleci community platform job instead of deleting it.

Reply:
I'd still like have to have a backup in case of a GCP region outage wherever the self-hosted runners are. We can change the tagging scheme for these two parallel workflows so they don't override each other, and perhaps temporarily copy the docker-bake?

Reply:
I think an entire GCP region outage is equally if not less likely than a circleci outage, to be honest. I doubt that circleci itself runs entirely multi-region. If you're concerned with a zone outage I can just make the GKE cluster multi-region  (which it's currently not) but I think even that one is very unlikely (in my 4 years of experience I never had a zone outage, only some specific service outages or degradations) and even if it happens it's not taking down testnet but just would add some roadblocks to land PRs in the few hours that the zone is down.

Reply:
I mean the question is really, how likely do you think a zone outage is in the next 1 week or so? I don't think we can or should build long term on both circle an GHA.

Reply:
It's more about the probability of outage of each independently, and having a backup already in place without having to ninja land a code change to get builds back. If it's easy to do, then why not have it temporarily until we make the more permanent change?

Reply:
I just don't like to have a bunch of temporary code - and potentially forgot to delete in time or confuse other devs which one is the "real one" to make a change - for things that are unlikely to happen or have a relatively simple mitigation. I think overall I just like to remove legacy things quickly when something new is being introduced, esp in cases there the change does not have high risk or causes disaster. Anyways, just my 2cents. I added the community platform build back to circle (hope circle has all the same envvars set) and will remove it next week then or so.